wp-autoupdates.php

```diff
@@ -87,6 +87,9 @@ function wp_autoupdates_selected_plugins( $update, $item ) {
  * Add autoupdate column to plugins screen.
  */
 function wp_autoupdates_add_plugins_autoupdates_column( $columns ) {
+	if ( ! current_user_can( 'update_plugins' ) || ! wp_autoupdates_is_plugins_auto_update_enabled() ) {
```

Comment:
We should be sure to keep the test here in sync with the early exit test in `wp_autoupdates_add_plugins_autoupdates_column_content()`.

Reply:
nice catch 👍